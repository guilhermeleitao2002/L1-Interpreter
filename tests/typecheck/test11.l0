type Option = union { Some: int, None: unit };
let opt = Some("string");
opt;;