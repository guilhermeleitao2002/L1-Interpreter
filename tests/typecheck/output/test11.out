Type Error:
TypeError: Variant type mismatch: expected int, got string